coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
But it's not clear how to make `[-a | -b] [-a | -c]`. `mutually_exclusive_group` only allows `[-a | -b | -c]`. I'll look into it more.